rbc/typesystem.py

```diff
@@ -1221,6 +1383,8 @@ def bits(self):
             return int(self[0][7:])
         if self.is_struct:
             return sum([m.bits for m in self])
+        if self.is_pointer:
+            return 64  # TODO: check that pointer type bitwidth is 64
```

Comment:
The LLVM datalayout has a [section](https://llvm.org/docs/LangRef.html#data-layout) that defines the size of a pointer:

> p[n]:\<size>:\<abi>:\<pref>:\<idx>
    This specifies the size of a pointer and its <abi> and <pref>erred alignments for address space n. The fourth parameter <idx> is a size of index that used for address calculation. If not specified, the default index size is equal to the pointer size. All sizes are in bits. The address space, n, is optional, and if not specified, denotes the default address space 0. The value of n must be in the range [1,2^23).

Reply:
Also, I guess one could include `sizeof(void*)` in https://github.com/omnisci/omniscidb-internal/pull/5276

Reply:
The alternative makes sense. If targets `sizeof(void*)` is not 64, the TargetInfo must provide `sizeof(void*)` information.

Reply:
Now using `return TargetInfo().sizeof('void*') or 64`